server/src/proxy/http/prom.rs

```diff
@@ -143,6 +188,35 @@ impl<Q: QueryExecutor + 'static> RemoteStorage for Proxy<Q> {
         ctx: &Self::Context,
         query: Query,
     ) -> StdResult<QueryResult, Self::Err> {
+        let metric = find_metric(&query.matchers)?;
+        let remote_req = PrometheusRemoteQueryRequest {
+            context: Some(ceresdbproto::storage::RequestContext {
+                database: ctx.schema.to_string(),
+            }),
+            query: query.encode_to_vec(),
+        };
+        if let Some(resp) = self
+            .maybe_forward_prom_remote_query(metric, remote_req)
```

Comment:
`Forward prom query` has already implemented in handle_prom_process_query.

Reply:
Explained in description.